code/game/objects/items/storage/firstaid.dm

```diff
@@ -62,6 +63,7 @@
 	name = "doctor's bag"
 	icon_state = "firstaid-surgeryalt"
 	item_state = "firstaid-surgeryalt"
+	worn_icon_state = "baguette"
```

Comment:
You turned my bag into a baguette.
I'll pay you double of what the French paid you to release my bag from the carbohydrate-prison.
(belt slot)

Reply:
@lowwy I didn't turn it into anything, rather doctors bags are supposed to have a worn state, but do not. 

I chose to use baguettes for several of these cases as they both have a valid back and waist sprite.

Whoever made doctors use "Doctor's Bags" at roundstart neglected their due diligence, it needs an actual worn sprite like the medkit doctors used before.

Reply:
Yeah no worries, just some banter, I though it would be something like that. just making sure!